Src/PRuntimes/PSymRuntime/pom.xml

```diff
@@ -428,7 +428,7 @@
     <maven.compiler.target>11</maven.compiler.target>
     <log4j2.configurationFile>${project.basedir}/src/main/resources/log4j2.xml</log4j2.configurationFile>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <revision>1.0.0</revision>
+    <revision>1.0.1</revision>
```

Comment:
Do we want to make this versioning consistent with the P versions? Just like we do it for all the other P nuget packages?

Reply:
Noted! We will do this when we make PSym release.